Classes/Emogrifier.php

```diff
@@ -397,6 +423,57 @@ public function removeUnprocessableHtmlTag($tagName)
     }
 
     /**
+     * Normalize the value of the "style" attribute and save it.
+     *
+     * @param $node
```

Comment:
Please add the type for the @param and the type hinting in the method declaration.